.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
comments in json aren't part of the standard so they're generally not supported and cause I/O failures, does vscode accept them?

Reply:
VSCode generated the comment, and accepts them, as I was able to run the command that this creates in VSCode